tools/classes.js

```diff
@@ -797,6 +797,7 @@ var classes = [
   'foam.nanos.crunch.CapabilityCategoryCapabilityJunction',
   'foam.nanos.crunch.CapabilityJunctionStatus',
   'foam.nanos.crunch.UserCapabilityJunction',
+  'foam.u2.wizard.data.BooleanHolder',
```

Comment:
Please remove from this particular PR.  The holders will change package.

Reply:
Addressed.